cli/src/commands/github/mongo.rs

```diff
@@ -0,0 +1,94 @@
+use mongodb::{Client as MongoClient, Collection, Cursor, Database};
+use mongodb::bson::doc;
```

Comment:
Removing mongo from the code is going to take a bit.

Reply:
I hope not. I was hoping maybe we could just move the code files over to Core as step 1.  We can worry about abstracting the Mongo interactions later and just focus on code organization this pass.

Reply:
Well, it kinda did take a bit because I had to: 
1. Implement my own Mongo Service (as prescribed) 
2. Use that Mongo service to do what I was doing, only now Mongo updates the entire document rather than just the last hash from GitHub.